src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,96 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface PropsTodoList {
+  todos: Todo[];
+  openModal(todo: Todo): void;
+  reset: Todo | null;
```

Comment:
checkout checklist's pt 3 about props naming
```suggestion
  todos: Todo[];
  openModal(todo: Todo): void;
  onReset: Todo | null;
```